app/jobs/onceoff/migrate_upload_extensions.rb

```diff
@@ -0,0 +1,11 @@
+module Jobs
+
+  class MigrateUploadExtensions < Jobs::Onceoff
+    def execute_onceoff(args)
+      Upload.all.each do |upload|
```

Comment:
This is going to 💥 on Discourses with a large number of uploads. It's better to use `Upload.find_each`